views/intercept/google-storage-warning.vm

```diff
@@ -0,0 +1,121 @@
+##
+#set ($rpContext = $profileRequestContext.getSubcontext('net.shibboleth.idp.profile.context.RelyingPartyContext'))
+#set ($username = $authenticationContext.getSubcontext('net.shibboleth.idp.authn.context.UsernamePasswordContext', true).getUsername())
+#set ($passwordEnabled = false)
+#if (!$passwordPrincipals or $passwordPrincipals.isEmpty() or $authenticationContext.isAcceptable($passwordPrincipals))
+  #set ($passwordEnabled = true)
+#end
+##
+<!DOCTYPE html>
+<html>
+  	<head>
+    	<meta charset="utf-8">
+<link rel="stylesheet" type="text/css" href="/css/opensans.css" media="screen, handheld" />
+<meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+<title>UW NetID sign-in</title>
+  <meta name="description" content="">
+  <meta name="author" content="">
+<meta name="viewport" content="width=device-width, initial-scale=1" />
+  <meta http-equiv="cleartype" content="on">
+<link rel="stylesheet" type="text/css" href="/css/weblogin-global.css" media="screen, handheld" />
+  <link rel="stylesheet" type="text/css" href="/css/weblogin-enhanced.css" media="screen  and (min-width: 40.5em)" />
+  <!--[if (lt IE 9)&(!IEMobile)]>
+  <link rel="stylesheet" type="text/css" href="/css/weblogin-enhanced.css" />
+  <![endif]-->
+
+
+  	</head>
+
+
+<body>
+
+  <h1 class="visuallyhidden">UW NetID sign-in</h1>
+
+  <div id="container">
+    <div id="main" role="main">
+
+      <h2 class="visuallyhidden" aria-flowto="weblogin_warning">Login</h2>
+      <div class="form">
+          <div><img src="/img/weblogin.png" height="57" width="198" alt="" aria-hidden="true"></div>
+
+          <form name="idpgooglestorageintercept" id="idpgooglestorageinterceptdiv" method="POST" action="$flowExecutionUrl" autocomplete="off">
+
+          <p/><strong><span data-contrast="auto">WARNING</span></strong>
+          <span data-contrast="auto">: UW Google 100GB storage limit enforcement begins June 11, 2024.</span>
+          
+          <p/>
+          Your UW Google account is currently approaching or over the 100GB storage limit.
+          Accounts that are at or over the 100GB limit will be restricted starting Tuesday June 11, 2024.
+          <p/>
+
+          All UW Google accounts in a restricted state are subject to permanent deletion.
+          Any associated Google content—including My Drive, Gmail, Sites, etc.—will be deleted.
+          <p/>
+
+          Preserve your UW Google data by
+          <a target="blank" href="https://itconnect.uw.edu/tools-services-support/software-computers/productivity-platforms/migrate-uw-google-or-uw-microsoft-content/migrate-uw-google-content/">migrating</a>
+          and/or deleting enough content to fall within the 100GB storage limit.
+          Please see the
+          <a target="blank" href="https://itconnect.uw.edu/tools-services-support/software-computers/productivity-platforms/google-productivity-platform/uw-google-changes-project/">UW Google Changes Project</a>
+          page for more information.
```

Comment:
`<p>` can't be a void element. Instead, it should paragraphs, like so:

```suggestion
          <p><strong><span data-contrast="auto">WARNING</span></strong>
          <span data-contrast="auto">: UW Google 100GB storage limit enforcement begins June 11, 2024.</span></p>

          <p>
          Your UW Google account is currently approaching or over the 100GB storage limit.
          Accounts that are at or over the 100GB limit will be restricted starting Tuesday June 11, 2024.
          </p>

          <p>
          All UW Google accounts in a restricted state are subject to permanent deletion.
          Any associated Google content—including My Drive, Gmail, Sites, etc.—will be deleted.
          </p>

          <p>
          Preserve your UW Google data by
          <a target="blank" href="https://itconnect.uw.edu/tools-services-support/software-computers/productivity-platforms/migrate-uw-google-or-uw-microsoft-content/migrate-uw-google-content/">migrating</a>
          and/or deleting enough content to fall within the 100GB storage limit.
          Please see the
          <a target="blank" href="https://itconnect.uw.edu/tools-services-support/software-computers/productivity-platforms/google-productivity-platform/uw-google-changes-project/">UW Google Changes Project</a>
          page for more information.
          </p>
```

Consider the above, or replacing `<p/>` with `<br />`